belgif-rest-problem-spring/src/test/java/io/github/belgif/rest/problem/spring/RoutingExceptionsHandlerTest.java

```diff
@@ -71,4 +78,27 @@ void handleHttpMessageNotReadable() {
         });
     }
 
+    @Test
+    void handleHttpRequestMethodNotSupported() {
+        ResponseEntity<Void> response = handler.handleHttpRequestMethodNotSupported(
+                new HttpRequestMethodNotSupportedException("POST", Arrays.asList("GET", "PUT")));
+        assertThat(response.getStatusCodeValue()).isEqualTo(405);
+        assertThat(response.getHeaders().getAllow()).containsExactly(HttpMethod.GET, HttpMethod.PUT);
+    }
+
+    @Test
+    void handleHttpMediaTypeNotAcceptable() {
+        ResponseEntity<Void> response = handler.handleHttpMediaTypeNotAcceptable(
+                new HttpMediaTypeNotAcceptableException(Collections.singletonList(MediaType.APPLICATION_JSON)));
+        assertThat(response.getStatusCodeValue()).isEqualTo(406);
+    }
+
+    @Test
+    void handleHttpMediaTypeNotSupported() {
+        ResponseEntity<Void> response = handler.handleHttpMediaTypeNotSupported(
```

Comment:
I don't understand. If read Peter, the specs state we should include the supported media type. We are not including because it's difficult to retrieve them in spring. But here, I see it's possible to get them from the exception. Should be not at least support if they are present ?

Reply:
Noted in https://github.com/belgif/rest-problem-java/issues/38#issuecomment-2379094962.

But in any case, the problem type and how to represent the supported media types in the problem body has not been standardized yet. So for now, a HTTP 406 with empty body should suffice (better than the current HTTP 500).